instance/tests/api/test_openedx_appserver.py

```diff
@@ -120,11 +131,14 @@ def test_get_details_permission_denied(self, username, message):
         self.assertEqual(response.data, {'detail': message})
 
     @patch_url(settings.OPENSTACK_AUTH_URL)
-    def test_get_details(self):
+    @ddt.data(
+        'user3', 'user4',
+    )
```

Comment:
I think we have to decide on a format for the parameters, on this file I see this two:

`
@ddt.data(
     'user3', 'user4',
)`

Reply:
Ok, I fixed this. I would be lucky if this were the biggest issue with the PR.
More changes pending.